src/vcpkg-configuration.json

```diff
@@ -0,0 +1,25 @@
+{
+  "overlay-triplets": [
+    "./triplets"
+  ],
+  "default-registry": {
+    "kind": "git",
+    "baseline": "b322364f06308bdd24823f9d8f03fe0cc86fd46f",
+    "repository": "https://github.com/microsoft/vcpkg"
+  },
+  "registries": [
+    {
+      "kind": "artifact",
+      "location": "https://github.com/microsoft/vcpkg-ce-catalog/archive/refs/heads/main.zip",
+      "name": "microsoft"
+    },
```

Comment:
Why is this registry needed here ? It seems out-of-date/unmaintained

Reply:
This is generated by default by vcpkg. I will check